packages/dd-trace/test/setup/core.js

```diff
@@ -5,6 +5,29 @@ const chai = require('chai')
 const sinonChai = require('sinon-chai')
 const proxyquire = require('../proxyquire')
 
+{
+  // get-port can often return a port that is already in use, thanks to a race
```

Comment:
@bengl Isn't EADDRINUSE thrown by the server when starting and not by `get-port`? If that is the case, then this PR doesn't really do anything.